client/src/components/ViewFilm.jsx

```diff
@@ -15,26 +19,32 @@ const ViewFilm = (props) => {
     //     toDelete(toDeleteContact)
     // }
 
-// Function to format the release date
+    // Function to format the release date
 
 
     return (
-        <Card>
-            <Card.Body>
-            <Card.Title>{props.film.name}</Card.Title>
-            <Card.Text>
-                <div>
-                    Released Date:{moment(props.film.released).format('LL')}
-                </div>
-                <div>
-                    Category:{props.film.category}
-                </div>
-                
-            </Card.Text>
-            {/* <Button variant="outline-danger" onClick={()=>{onDelete(contact)}} style={{padding: '0.6em', marginRight:'0.9em'}}><ioicons.IoTrash/></Button>
-            <Button variant="outline-info" onClick={()=>{onUpdate(contact)}} style={{padding: '0.6em'}}> <ioicons.IoSync/></Button> */}
-            </Card.Body>
-        </Card>
+        <>
```

Comment:
What's the purpose of this Fragment? It looks like it just contains one element, so I would expect the behavior to be the same without it?

Reply:
Do you mean the card element?

Reply:
Ah, I mean the empty `<>` (https://react.dev/reference/react/Fragment)